test/client/src/lib.rs

```diff
@@ -128,20 +136,59 @@ impl DefaultTestClientBuilderExt for TestClientBuilder {
 }
 
 fn genesis_config(changes_trie_config: Option<ChangesTrieConfiguration>) -> GenesisConfig {
-	GenesisConfig::new(
-		changes_trie_config,
-		vec![
-			sr25519::Public::from(Sr25519Keyring::Alice).into(),
-			sr25519::Public::from(Sr25519Keyring::Bob).into(),
-			sr25519::Public::from(Sr25519Keyring::Charlie).into(),
-		],
-		vec![
-			AccountKeyring::Alice.into(),
-			AccountKeyring::Bob.into(),
-			AccountKeyring::Charlie.into(),
-		],
-		1000,
-		Default::default(),
-		Default::default(),
+	cumulus_test_service::local_testnet_genesis(changes_trie_config)
+}
+
+fn additional_storage_with_genesis(genesis_block: &Block) -> BTreeMap<Vec<u8>, Vec<u8>> {
+	map![
+		twox_128(&b"latest"[..]).to_vec() => genesis_block.hash().as_fixed_bytes().to_vec()
+	]
+}
+
+/// Transfer some token from one account to another using a provided test `Client`.
+pub fn transfer(
+	client: &Client,
+	origin: sp_keyring::AccountKeyring,
+	dest: sp_keyring::AccountKeyring,
+	value: Balance,
+) -> UncheckedExtrinsic {
+	let function = Call::Balances(pallet_balances::Call::transfer(dest.public().into(), value));
+
+	let current_block_hash = client.info().best_hash;
+	let current_block = client.info().best_number.saturated_into();
+	let genesis_block = client.hash(0).unwrap().unwrap();
+	let nonce = 0;
+	let period = BlockHashCount::get()
+		.checked_next_power_of_two()
+		.map(|c| c / 2)
+		.unwrap_or(2) as u64;
+	let tip = 0;
+	let extra: SignedExtra = (
+		frame_system::CheckSpecVersion::<Runtime>::new(),
+		frame_system::CheckGenesis::<Runtime>::new(),
+		frame_system::CheckEra::<Runtime>::from(Era::mortal(period, current_block)),
+		frame_system::CheckNonce::<Runtime>::from(nonce),
+		frame_system::CheckWeight::<Runtime>::new(),
+		pallet_transaction_payment::ChargeTransactionPayment::<Runtime>::from(tip),
+	);
+	let raw_payload = SignedPayload::from_raw(
+		function.clone(),
+		extra.clone(),
+		(
+			VERSION.spec_version,
+			genesis_block,
+			current_block_hash,
+			(),
+			(),
+			(),
+		),
+	);
+	let signature = raw_payload.using_encoded(|e| origin.sign(e));
+
+	UncheckedExtrinsic::new_signed(
+		function.clone(),
+		origin.public().into(),
+		Signature::Sr25519(signature.clone()),
+		extra.clone(),
 	)
```

Comment:
Could you put this into a generic function? I like the `transfer` function and it could call this new function. However, we can later use this for other transactions.

Reply:
Done in d00c4db83a9d8ea55955d045f13e1c5f4a2d8082